### PR TITLE
add functionality to remove illegal project name characters

### PR DESCRIPTION
**Pull request by josh-chou**

### CODE REVIEW

File: service/resources/bluebeam.py [hunk 1]
```diff
@@ -182,6 +182,17 @@ def create_project(access, project_name):
     print("Created project id:{0}".format(idee))
     return idee
 
+def sanitize_project_name(project_name):
+    """
+        returns new project name after illegal chars have been removed
+        https://support.bluebeam.com/articles/unsupported-characters-in-studio/
+    """
+    #allow alpha numeric, underscore, period, spaces
+    new_name = re.sub(r"[^\w. #]", "", project_name)
```
Comment (hshaosf) [inline service/resources/bluebeam.py:191]: What do you think of this suggestion? does \w include dashes? 
```suggestion
    new_name = re.sub(r"[^\w. #]", " ", project_name)
```

**Review by hshaosf — APPROVED** [review]
in general, probably add more assert test cases such as dashes? and special characters in between "valid" characters. other LGTM.

File: service/resources/bluebeam.py (hunk 1 above)
Comment (josh-chou) [reply service/resources/bluebeam.py:191]: You want to replace illegal chars with spaces?  In the case of "TEST 1 SOUTH VAN NESS *", that will result with a project name with trailing space.
\w includes _ but not -

Comment (hshaosf) [reply service/resources/bluebeam.py:191]: then for trailing and possibily leading space, maybe we can do a trim?

Comment (josh-chou) [reply service/resources/bluebeam.py]: https://github.com/SFDigitalServices/bluebeam-microservice/pull/53/commits/aefe02a81990ff107022cd0184f4676bd4d4d440

**Pull request merged**